README.md

```diff
@@ -2,157 +2,133 @@
 page_type: sample
 languages:
 - javascript
+- nodejs
 products:
-- azure
-description: "For Service-to-Azure-Service authentication, the approach so far involved creating an Azure AD application and associated credential,"
-urlFragment: app-service-msi-keyvault-node
+- azure-key-vault
+- azure-app-service
+description: "How to set and get secrets from Azure Key Vault with Azure Managed Identities and Node.js."
+urlFragment: get-set-keyvault-secrets-managed-id-nodejs
 ---
 
-# Use Key Vault from App Service with Managed Service Identity and Nodejs
+# How to set and get secrets from Azure Key Vault with Azure Managed Identities and Node.js
 
-## Background
-For Service-to-Azure-Service authentication, the approach so far involved creating an Azure AD application and associated credential, and using that credential to get a token. While this approach works well, there are two shortcomings:
-1. The Azure AD application credentials are typically hard coded in source code. Developers tend to push the code to source repositories as-is, which leads to credentials in source.
-2. The Azure AD application credentials expire, and so need to be renewed, else can lead to application downtime.
+## SDK Versions
+In this sample, you will find the following folders:
+* **v3** - references Key Vault SDK v3
+* **v4** - references Key Vault SDK v4
 
-With [Managed Service Identity (MSI)](https://docs.microsoft.com/en-us/azure/app-service/app-service-managed-service-identity), both these problems are solved. This sample shows how a Web App can authenticate to Azure Key Vault without the need to explicitly create an Azure AD application or manage its credentials. 
+## Introduction
+This sample will show how Web App get a secret at runtime from Azure Key Vault using developer account during development, and using Azure Managed Identities when deploy to Azure, without any code change between local development environment and Azure. As a result, it doesn't have to explicitly handle a service principal credential to authenticate to Azure AD to get a token to call Key Vault. It doesn't have to worry about renewing the service principal credential either, since Azure Managed Identities takes care of that.
 
->Here's another sample that how to use MSI from inside an Azure VM with a Managed Service Identity (MSI) - [https://github.com/Azure-Samples/resource-manager-node-manage-resources-with-msi](https://github.com/Azure-Samples/resource-manager-node-manage-resources-with-msi)
 
 ## Prerequisites
 To run and deploy this sample, you need the following:
-1. An Azure subscription to create an App Service and a Key Vault. 
+* [Node.js]
+* [Git]
+* An Azure subscription to create a Key Vault and other services(such as App Service) used in this sample.
```

Comment:
An Azure subscription to create a Key Vault and other services, such as App Service, used in this sample.